packages/dds/tree/src/test/utils.ts

```diff
@@ -109,16 +119,31 @@ export class TestTreeProvider {
         ));
     }
 
+    /**
+     * Give this {@link TestTreeProvider} the ability to summarize on demand during a test by creating a summarizer
+     * client for the container at the given index.  This can only be called when the summarizeOnDemand field is
+     * set to true.
+     * @returns void after a summary has been resolved. May be called multiple times.
+     */
+    public async summarize(): Promise<void> {
+        if (TestTreeProvider.summarizer !== undefined) {
```

Comment:
This should throw an error if the summarizer is undefined, rather than silently doing nothing (see related comment in `create()`

Reply:
Added assert statement to throw error when summarizer is undefined